tomcat/detect.go

```diff
@@ -92,3 +97,17 @@ func (d Detect) Detect(context libcnb.DetectContext) (libcnb.DetectResult, error
 	result.Plans[0].Provides = append(result.Plans[0].Provides, libcnb.BuildPlanProvide{Name: PlanEntryJVMApplicationPackage})
 	return result, nil
 }
+
+func containsWarFiles(dir string) (bool, error) {
+	files, err := ioutil.ReadDir(dir)
```

Comment:
```suggestion
	files, err := os.ReadDir(dir)
```

`ioutil` is deprecated.

Reply:
@dmikusa, thanks a  lot for pointing it out. I have removed it and moved `containsWarFiles` to a internal package `internal.util`